internal/integration/assert.go

```diff
@@ -107,7 +107,7 @@ func AssertListDetails[L ListDetails, D ListDetailsMsg[L]](t assert.TestingT, ex
 	if wantDetails.GetTimestamp() != nil {
 		gotCD := gotDetails.GetTimestamp().AsTime()
 		wantCD := time.Now()
-		assert.WithinRange(t, gotCD, wantCD.Add(-1*time.Minute), wantCD.Add(time.Minute))
+		assert.WithinRange(t, gotCD, wantCD.Add(-10*time.Minute), wantCD.Add(time.Minute))
```

Comment:
do we actually need 10 minutes... feels like we could also just omit that test

Reply:
I changed it to 1 minute in the first PR, and saw that the pipeline failed once because the start of the test and the timing of the check were more than 1 minute apart.